apps/arweave/src/ar_data_sync.erl

```diff
@@ -944,7 +944,6 @@ handle_cast({store_fetched_chunk, Peer, Time, TransferSize, Byte, Proof} = Cast,
 							ar_util:cast_after(1000, self(), Cast),
 							{noreply, State};
 						false ->
-							ar_events:send(peer, {served_chunk, Peer, Time, TransferSize}),
```

Comment:
Where do we record chunk download statistics now?

Reply:
Previously the only action we did in response to a `peer, served_chunk` event was to update the peer rating in `ar_peers` (that was the main reason I removed the event).

Now the rating is updated [here](https://github.com/ArweaveTeam/arweave/pull/443/files/b32f7c4ee9411df9b93358ca80eb6a2dbfb0a99c#diff-f2d3cef82c02c5e4782cac6bd44f38f590355bb8573c4c980198ee030ccc43bcR320)
 in `ar_data_sync_worker_master` whenever an `ar_data_sync_worker` completes syncing a chunk.
 
 Other statistics are also update in `ar_data_sync_worker_master` whenever an `ar_data_sync_worker` [completes syncing a range](https://github.com/ArweaveTeam/arweave/pull/443/files/b32f7c4ee9411df9b93358ca80eb6a2dbfb0a99c#diff-f2d3cef82c02c5e4782cac6bd44f38f590355bb8573c4c980198ee030ccc43bcR109)